bundle/artifacts/prepare.go

```diff
@@ -35,6 +36,16 @@ func (m *prepare) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics
 
 	for _, artifactName := range utils.SortedKeys(b.Config.Artifacts) {
 		artifact := b.Config.Artifacts[artifactName]
+		if artifact == nil {
+			l := b.Config.GetLocation("artifacts." + artifactName)
+			diags = append(diags, diag.Diagnostic{
+				Severity:  diag.Error,
+				Summary:   "Artifact not properly configured",
+				Detail:    "please specify artifact properties",
+				Locations: []dyn.Location{l},
+			})
+			continue
+		}
```

Comment:
@andrewnester For empty artifacts (ie. `{}`), we return a different error (see L87).

Can we consolidate?